.gitignore

```diff
@@ -13,3 +13,4 @@ vendor/
 
 /.env*
 /docker-compose.*.yml
+storage/
```

Comment:
This is a minor quibble, but it should technically be `/storage/`, if I understand it correctly. :) But I think it's incorrect elsewhere, too (like for Transformer PSC). I don't mind if this is changed or not.